webpack/rules/scss.js

```diff
@@ -31,6 +31,7 @@ export default [
           postcssOptions: {
             plugins: () => [
               autoprefixer(AUTOPREFIXER),
+              require('postcss-import')
```

Comment:
I set the same string, but problem does not disappear)

Reply:
idk what`s the problem, but for me it works as in the attached screenshot. maybe try to restart your de

Reply:
ok, I'll check it again

Reply:
![terminal](https://user-images.githubusercontent.com/61930014/109161020-6cf18a00-772b-11eb-8777-61081cfd3698.png)

I was checking in the terminal (without code editors). First off, deleted **node_modules**, but the same mistake

Reply:
checked this also, and you are right, even without this line after caching (after first build) this message disappears, so this change was useless after all